grakn-engine/src/main/java/ai/grakn/engine/SystemKeyspace.java

```diff
@@ -157,18 +157,20 @@ public boolean deleteKeyspace(String keyspace){
      * multiple times.
      */
     public void loadSystemOntology() {
+        Stopwatch timer = Stopwatch.createStarted();
         try (GraknGraph graph = factory.getGraph(SYSTEM_GRAPH_NAME, GraknTxType.WRITE)) {
             if (graph.getOntologyConcept(KEYSPACE_ENTITY) != null) {
                 checkVersion(graph);
                 return;
             }
+            LOG.info("No other version found, loading ontology for version {}", GraknVersion.VERSION);
             loadSystemOntology(graph);
             graph.getResourceType(SYSTEM_VERSION).putResource(GraknVersion.VERSION);
             graph.admin().commitNoLogs();
-            LOG.info("Loaded system ontology to system keyspace.");
-        } catch (InvalidGraphException | NullPointerException e) {
-            e.printStackTrace(System.err);
-            LOG.error("Could not load system ontology. The error was: " + e);
+            LOG.info("Loaded system ontology to system keyspace. Took: {}", timer.stop());
```

Comment:
I bet you most of the time is spent in initializing the keyspace. This is not related to this PR but I thought we should note it here.